README.md

```diff
@@ -47,6 +47,9 @@ run style checks use `composer cs`.
 ### Version 2.2.0 (dev)
 
 * Changed the minimum PHP version to 7.0
+* Added `campaign_code` field to `subscriptions` table. The campaign code is to differentiate between "remind me later" 
+  and other usages of the subscription functionality. The campaign code can be different from the "campaign" in the 
```

Comment:
At the moment, it's just "Remind me later" and "other", but there might be more usages in the future so a boolean flag seems to be inadvisable.